Sources/RemoteMessaging/RemoteMessagingStore.swift

```diff
@@ -60,61 +73,77 @@ public final class RemoteMessagingStore: RemoteMessagingStoring {
         self.notificationCenter = notificationCenter
         self.errorEvents = errorEvents
         self.remoteMessagingAvailabilityProvider = remoteMessagingAvailabilityProvider
+        self.context = database.makeContext(concurrencyType: .privateQueueConcurrencyType, name: Constants.privateContextName)
 
         featureFlagDisabledCancellable = remoteMessagingAvailabilityProvider.isRemoteMessagingAvailablePublisher
-            .map { !$0 }
+            .filter { !$0 }
```

Comment:
This was a clear bug, unrelated to the pixels increase. We want to only filter false values, not invert them.